### ISSUE TITLE
Player order indicator

### ISSUE THREAD

**Issue by d-albrecht** [Sesu8642/FeudalTactics#32]
What if you could see the order of the players somewhere on the screen?

The bottom is already taken and the top is quite crowded as well, so I'd suggest to have a bar at either side that indicates (a) who had the first turn, (b) whose turn it currently is, and (c) who will come next

The top position would always be the player with the first turn. Then all the other players in order. This should be visible on the map selection screen already, so that you know who would act before you even got to play. The current player could be indicated by an arrow, or a border around the color or something else. If one player is no longer participating, you could either remove the color from the bar, or cross it out, or whatever.

You could even use this to indicate the state of the game. You know how Slay has bar charts to indicate the number of tiles each color has? You could also include that into the side bar as well, in terms of sprites. If there is a gravestone on your color, that means you are out. if there is a tree on your color, then you have a very weak position, if you have a peasant, you have some tiles, if you have a baron on your color, you have (almost) wiped out everyone else.

That's a quick draft of what I imagine (without the sprites):
![Order_Indicator](https://user-images.githubusercontent.com/5196458/194863547-556e6e6e-d58f-4e86-9200-02e1f330b3d9.png)

**Comment by Sesu8642** [comment]
Good idea. The placement surely works for the desktop version but could be annoying on mobile if playing in portrait mode. Ideally, it would move somewhere else or get hidden if the window becomes too small. But that's probably difficult with libGDX if possible at all.

**Comment by Sesu8642** [comment]
There could be a human or robot icon inside or instead of the tiles to indicate the player type.

**Comment by d-albrecht** [comment]
Especially if it ever comes to multi-player games (or if you offer the option to switch the human player's color), this could be a useful idea, too.
